mrtt-ui/src/components/SiteBackgroundForm.js

```diff
@@ -0,0 +1,290 @@
+          <Controller
+            name='lawStatus'
+            control={control}
+            defaultValue=''
+            render={({ field }) => (
+              <TextField {...field} select value={field.value} label='select'>
+                {questions.lawStatus.options.map((item, index) => (
+                  <MenuItem key={index} value={item}>
+                    {item}
+                  </MenuItem>
+                ))}
+              </TextField>
+            )}
+          />
+          <ErrorText>{errors.lawStatus?.message}</ErrorText>
+        </FormQuestionDiv>
+        {/* Management Area*/}
+        <FormQuestionDiv>
+          <FormLabel>{questions.managementArea.question}</FormLabel>
+          <Controller
+            name='managementArea'
+            control={control}
+            defaultValue=''
+            render={({ field }) => <TextField {...field} value={field.value}></TextField>}
+          />
+          <ErrorText>{errors.managementArea?.message}</ErrorText>
+        </FormQuestionDiv>
+        {/* Protection Status*/}
+        <CheckboxGroupWithLabelAndController
+          fieldName='protectionStatus'
+          reactHookFormInstance={reactHookFormInstance}
+          options={questions.protectionStatus.options}
+          question={questions.protectionStatus.question}
+          shouldAddOtherOptionWithClarification={true}
+        />
+        <ErrorText>{errors.protectionStatus?.selectedValues?.message}</ErrorText>
+        {/* areStakeholdersInvolved */}
+        <FormQuestionDiv>
+          <FormLabel>{questions.areStakeholdersInvolved.question}</FormLabel>
+          <Controller
+            name='areStakeholdersInvolved '
+            control={control}
+            defaultValue=''
+            render={({ field }) => (
+              <TextField {...field} select value={field.value} label='select'>
+                {questions.areStakeholdersInvolved.options.map((item, index) => (
+                  <MenuItem key={index} value={item}>
+                    {item}
+                  </MenuItem>
+                ))}
+              </TextField>
+            )}
+          />
+          <ErrorText variant='subtitle' sx={{ color: 'red' }}>
+            {errors.areStakeholdersInvolved?.message}
+          </ErrorText>
+        </FormQuestionDiv>
+        {/* Government Arrangement */}
+        <FormQuestionDiv>
+          <FormLabel>{questions.govermentArrangement.question}</FormLabel>
+          <Controller
+            name='governmentArrangement'
+            control={control}
+            defaultValue={[]}
+            render={({ field }) => (
+              <Select
+                {...field}
+                multiple
+                value={field.value}
+                label='select'
+                input={<OutlinedInput id='select-multiple-chip' label='Chip' />}
+                renderValue={(selected) => (
+                  <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 0.5 }}>
```

Comment:
Id normally advocate for the maps being made into functions that return jsx fragments, and avoiding inline styling in practice, but meh. I think its one of those corners we can cut for timelines.